ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/event_handlers.schema.yml

```diff
@@ -42,6 +42,72 @@ keys:
           - on-boot
           - on-logging
           - on-startup-config
+          - on-maintenance
+        trigger_on_maintenance:
+          description: Settings required for trigger 'on-maintenance'.
+          type: dict
+          keys:
+            operation:
+              type: str
+              required: true
+              valid_values:
+              - "enter"
+              - "exit"
+            bgp:
+              description: Trigger condition occurs on maintenance operation of specified BGP peer.
+              type: dict
+              keys:
+                neighbor_address_ipv4:
+                  type: str
+                neighbor_address_ipv6:
+                  type: str
+                peer_group_name:
+                  type: str
+            stage:
+              type: str
+              required: true
+              valid_values:
+              - after
+              - before
+              - all
+              - begin
+              - end
+              - vrf
+            after_before_stage:
+              type: str
+              description: Action is triggered after/before specified stage.
+              valid_values:
+              - "bgp"
+              - "linkdown"
+              - "mlag"
+              - "ratemon"
+            vrf:
+              type: str
+              description: VRF name.
+            interface:
+              description: Trigger condition occurs on maintenance operation of specified interface.
+              type: dict
+              keys:
+                interface_type:
+                  type: str
+                  required: true
+                  valid_values:
+                  - Ethernet
+                  - Fabric
+                  - Loopback
+                  - Management
+                  - Port-Channel
+                  - Tunnel
+                  - Vlan
+                  - Vxlan
+                interface_number:
+                  type: int
+                  convert_types:
+                  - str
+                  required: true
```

Comment:
You can just make interface a string. CLI accepts "Ethernet1" like we have everywhere else.

Reply:
Updated!